playwright-e2e/playwright.config.ts

```diff
@@ -69,7 +69,7 @@ const testConfig: PlaywrightTestConfig = {
     browserName: 'chromium',
     headless: true,
     launchOptions: {
-      slowMo: TEST_SLOW_MO ? parseInt(TEST_SLOW_MO) : 500,
+      slowMo: TEST_SLOW_MO ? parseInt(TEST_SLOW_MO) : 100,
```

Comment:
making tests run slower may be the root causes for sporadic waitForResponse failures. testing with a smaller delay to see if it helps.